src/clustering/administration/main/command_line.cc

```diff
@@ -1178,6 +1478,7 @@ void get_rethinkdb_serve_options(std::vector<options::help_section_t> *help_out,
     help_out->push_back(get_help_options(options_out));
     help_out->push_back(get_log_options(options_out));
     help_out->push_back(get_config_file_options(options_out));
+    help_out->push_back(get_tls_options(options_out));
```

Comment:
Suggestion: Move this below the `get_network_options` above. That way the network options will be printed next to the TLS options.